stripe/src/main/java/com/stripe/android/view/MaskedCardView.java

```diff
@@ -80,6 +80,19 @@ public MaskedCardView(Context context, AttributeSet attrs, int defStyle) {
         init();
     }
 
+    @Override
+    public boolean isSelected() {
+        return mIsSelected;
+    }
+
+    @Override
+    public void setSelected(boolean selected) {
```

Comment:
This isn't going to be public -- only listed as that now because I have it in the example application. (My future PR in the example application will remove this)